File: scripts/algorithms/U/Unique Paths/Unique Paths.js
// Runtime: 62 ms (Top 95.54%) | Memory: 42.3 MB (Top 53.45%)
var uniquePaths = function(m, n) {
    let count = Array(m)
    for(let i=0; i<m; i++) count[i] = Array(n)
    for(let i=0; i<m; i++){
        for(let j=0; j<n; j++){
            if(i == 0 || j == 0) count[i][j] = 1
            else count[i][j] = count[i][j-1] + count[i-1][j]
        }
    }
    return count[m-1][n-1]
};

File: scripts/algorithms/U/Unique Paths/Unique Paths.rs
// Runtime: 2 ms (Top 20.53%) | Memory: 2.00 MB (Top 53.64%)

impl Solution {
    pub fn unique_paths(m: i32, n: i32) -> i32 {
        let mut ans: i64 = 1;
        for i in 1..=m as i64 - 1 {
            ans = ans * (n as i64 - 1 + i) / i;
        }
        ans as i32
    }
}
